pages/VerifiableCredentials/Types/DID.md

```diff
@@ -0,0 +1,80 @@
+# DIDs
+
+DSNP Users are referenced in Verifiable Credentials documents via DIDs compliant with the [Decentralized Identifiers v1.0](https://www.w3.org/TR/did-core/) specification.
+
+Applications that make use of Verifiable Credentials issued by DSNP Users MUST be able to resolve DSNP User DIDs to DID documents in order to verify that signatures were created with keys controlled by that user.
+A DSNP DID document is effectively a document aggregating DSNP User Data.
+Compliant DSNP systems are encouraged to provide their own DID resolver libraries.
+
+## DID Method Syntax
+
+A DSNP DID uses `dsnp` (lowercase) as the method name and the DSNP User Id as the method-specific identifier, as the following example illustrates:
+
+`did:dsnp:123456`
+
+This DID identifies the DSNP User with User Id 123456.
+In this format, the DSNP User Id is serialized in decimal form with no additional punctuation. 
+
+References to identifiers within a DID document are formed by appending URL fragments to a DID.
+For example, a Verifiable Credential might reference the public key to be used to verify a document's signature as `did:dsnp:123456#key1`, assuming the document included a `verificationMethod` with `"id": "key1"`.
+
+## DID Document
+
+A DSNP DID document is a JSON-LD document representing key material associated with a DSNP User.
+
+| Property | Required | JSON Type | Description | Restrictions |
+| --- | --- | --- | --- | --- |
```

Comment:
YES and NO should be YES and no to be consistent with other sections of the documentation.